discretize/View.py

```diff
@@ -777,8 +777,7 @@ def plot_3d_slicer(self, v, xslice=None, yslice=None, zslice=None,
         # Connect figure to scrolling
         fig.canvas.mpl_connect('scroll_event', tracker.onscroll)
 
-        # Show figure
-        plt.show()
+        return fig
```

Comment:
cc @prisae: just a small functionality update here I wanted to bring you in on. It seems like there are some use-cases where having access to the figure handle outside would be handy if it is generated inside this function. Does this work for you?

Reply:
Yes, no problem, I don't have any hard feelings with regards to how this is handled. I did not overly adjust the `plot_3d_slicer` from my usage/style to SimPEG-style. I think the idea  was we just bring it in, and then adjust it where necessary, so perfectly fine by me.